.github/workflows/e2e-tests.yaml

```diff
@@ -64,7 +64,8 @@ jobs:
       matrix:
         cluster-type: ["eksctl", "kops"]
         arch: ["x86", "arm"]
-        family: ["AmazonLinux2", "Bottlerocket"]
+        family:
+          ["AmazonLinux2", "AmazonLinux2023", "Bottlerocket", "Ubuntu2004"]
```

Comment:
We should revert this change probably. See the comment regarding Ubuntu2004 and Ubuntu2204 in a few lines below